source/js/utils.js

```diff
@@ -290,17 +290,6 @@ NexT.utils = {
     });
   },
 
-  getComputedStyle: function(element) {
```

Comment:
I remember that I wrote this `getComputedStyle` function because the `display: none` style was set, and the value of `siteNav.scrollHeight` was always zero. @PaperStrike Your implementation is more concise 👍